truffle.js

```diff
@@ -1,9 +1,63 @@
+/**
```

Comment:
I've been meaning to make this its own package on npm / etc. so we don't have to copy it around all the time. We can replace it with the package when we do move this out.

The vision for sharing this particular config is to have a generic Aragon-specific truffle config that looks up any hot keys you might want to specify for local publishing / etc. in the same directory (`~/.aragon/<network>_key.json`).

Reply:
Make lot of sense, I'll keep it in mind